src/chapters/data-and-variables/values-and-data-types.rst

```diff
@@ -92,35 +103,57 @@ JavaScript doesn't care whether you use single or double quotes to surround your
+When you type a large integer value, you might be tempted to use commas between groups of three digits, as in ``42,000``. This is not a legal integer in JavaScript, but it does mean something else, which is legal:
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42000);
     console.log(42,000);
 
+**Output**
+
+::
+
+   42000
+   42 0
 
-Well, that's not what we expected at all! Because of the comma, JS chose to treat this as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
+Well, that's not what we expected at all! Because of the comma, JavaScript chose to treat ``42,000`` as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42, 17, 56, 34, 11, 4.35, 32);
     console.log(3.4, "hello", 45);
 
-Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intended.
+**Output**
+
+::
+
+   42 17 56 34 11 4.35 32
+   3.4 'hello' 45
+
+Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intend.
 
 Type Systems
 ------------
 
 .. index:: ! type system
 
-Every programming langauge has a **type system**, which is the set of rules that determine how it deals with data of different types. In particular, how values are divided up into different data types of one characteristic of a type system.
+Every programming langauge has a **type system**, which is the set of rules that determine how it deals with data of different types. In particular, how values are divided up into different data types is one characteristic of a type system.
 
 In many programming languages, integers and floats are considered to be different data types. For example, in Python ``42`` is of the ``int`` data type, while ``42.0`` is of the ``float`` data type.
 
-When discussing the differences between programming languages, the details of tyep systems are one of the main things that programmers consider. There are other aspects of type systems beyond just how values are categorized. We will explore these in future lessons.
+.. note:: While JavaScript does not distinguish between floats and integers, at times we may wish to do so in our programs. For example, an inventory-tracking program might not want insist that counts of items are integers.
```

Comment:
The second sentence does not make sense.

Reply:
Also, line 121 contains a typo.  Replace 'tyep' with 'type'.

Reply:
New second sentence: "For example, consider an inventory-tracking program that stores items along with the number of each item in stock. It would not make sense for there to be float quantities of items (for example, 3.5 shirts), so the program may wish to only allow quantities to be stored as integers."

Better?

Reply:
That still feels very wordy. I think you can bring it down even further to: "For example, an inventory-tracking program stores items and the number of each number in stock. Since a store cannot have 3.5 shirts in stock, the programmer makes the quantity of each item integer values as opposed to floats."